tests/meta/archive/conftest.py

```diff
@@ -48,6 +48,12 @@ def ok_ddn_response(ok_response: Response):
     return ok_response
 
 
+@pytest.fixture(name="ok_ddn_job_status_response")
+def fixture_ok_ddn_job_status_response(ok_response: Response):
```

Comment:
Remove prefix `fixture` from the function name and the name parameter from the fixture decorator. Same for all the new fixtures in this file